python/python/glide/async_commands/core.py

```diff
@@ -5518,6 +5518,86 @@ async def getex(
             await self._execute_command(RequestType.GetEx, args),
         )
 
+    async def dump(
+        self,
+        key: str,
+    ) -> Optional[bytes]:
+        """
+        Serialize the value stored at `key` in a Valkey-specific format and return it to the user.
+
+        See https://valkey.io/commands/dump for more details.
+
+        Args:
+            key (str): The `key` of the set.
```

Comment:
As far as I can tell the key may not be a set right?
```suggestion
            key (str): The `key` to serialize.
```